app/models/actor_attributes.rb

```diff
@@ -209,8 +226,17 @@ def sustainability
   end
 
   def companies_exporting
+    quant = Quant.find_by(:name => 'SOY_')
+
+    unit = quant.unit
+    value_divisor = 1
+    if (unit.casecmp('tn') == 0)
```

Comment:
This feels like a less hacky hack than simply dividing by 1k all the time.

Reply:
yes